demo/GlutConfig.cmake

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required (VERSION 3.1)
+
+#Configure GLUT
+set(FREEGLUT_BUILD_DEMOS OFF CACHE BOOL "Build FreeGLUT demos.")
+set(FREEGLUT_BUILD_SHARED_LIBS OFF CACHE BOOL "Build FreeGLUT shared library.")
+set(FREEGLUT_PRINT_ERRORS OFF CACHE BOOL "Lib prints errors to stderr")
+set(FREEGLUT_PRINT_WARNINGS OFF CACHE BOOL "Lib prints warnings to stderr")
+
+#Set GLUT variables
+set(GLUT_PATH
+    ${CMAKE_CURRENT_SOURCE_DIR}/../third_party/FreeGLUT/freeglut/freeglut
+)
+set(GLUT_LIBRARY_DIRS
+    ${CMAKE_CURRENT_BINARY_DIR}/third_party/FreeGLUT/freeglut/freeglut
+)
+set(GLUT_INCLUDE_DIRS
+    ${CMAKE_CURRENT_SOURCE_DIR}/../third_party/FreeGLUT/freeglut/freeglut/include
+)
+
+add_subdirectory(${GLUT_PATH} ${GLUT_LIBRARY_DIRS})
```

Comment:
Okay, I could move `add_subdirectory` to the `CMakeLists.txt`. 
However, I would still need to add the second argument anyway, since glut directory is not an actual subdirectory, without it would get the following error message from CMake: `add_subdirectory not given a binary directory but the given source directory...`. I could make a symlink, though.